pkg/apb/client.go

```diff
@@ -185,16 +171,13 @@ func (c *Client) PullImage(imageName string) error {
 	return nil
 }
 
-func (c *Client) refreshLoginToken(clusterConfig ClusterConfig) error {
-	c.log.Debug("Refreshing login token...")
-	c.log.Debug("target: [ %s ]", clusterConfig.Target)
-	c.log.Debug("user: [ %s ]", clusterConfig.User)
-	c.log.Debug("password:[ %s ]", clusterConfig.Password)
+func (c *Client) Login() error {
+	c.log.Debug("Logging in to openshift...")
 
 	output, err := runCommand(
-		"oc", "login", "--insecure-skip-tls-verify", clusterConfig.Target,
-		"-u", clusterConfig.User,
-		"-p", clusterConfig.Password,
+    "oc", "login", "https://kubernetes.default",
+    "--certificate-authority", "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+    "--token", "$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)",
```

Comment:
Moved the serviceaccount login logic to the broker init, since it only needs to happen once, and I just check if the openshift target is specified, and if it isn't try to log in with the service account.